sonar-slang-plugin/src/main/java/com/sonarsource/slang/kotlin/KotlinCommentAnalyser.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * SonarSource SLang
+ * Copyright (C) 2009-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package com.sonarsource.slang.kotlin;
+
+import com.sonarsource.slang.api.ASTConverter;
+import com.sonarsource.slang.api.BlockTree;
+import com.sonarsource.slang.api.IdentifierTree;
+import com.sonarsource.slang.api.NativeTree;
+import com.sonarsource.slang.api.TextPointer;
+import com.sonarsource.slang.api.TextRange;
+import com.sonarsource.slang.api.Tree;
+import com.sonarsource.slang.api.TreeMetaData;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'com.sonarsource.slang.api.TreeMetaData'. ([squid:UselessImportCheck](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.slang%3Aslang&pullRequest=127&issues=AWRbS0Hp-1T-AjkxlrYF&open=AWRbS0Hp-1T-AjkxlrYF)

[ISSUE_KEY]: AWRbS0Hp-1T-AjkxlrYF